### DISCUSSION TITLE
[Kavita] Improve recognition of names in series folders under root folders

### DISCUSSION THREAD

**Discussion by ytoaa** [Kareadita/Kavita#2930] [Ideas]
Current Kavita systems also try to analyze the names of folders under the root folder and divide them into chapters, volumes, and specials.
This tends to slow down the overall execution
Affects the series name itself.
I want to see the name of the folder under the root folder itself
It's recognized as a series name
If a parent series name exists in a subfolder, it would be a good idea to exclude that string from the chapter, volume, and special classification.

This is an archive type of user, and in particular
Largely affects users of the image type.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

2y

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
I'm trying to understand what you're asking from this FR. Are you asking for better recognition support for the Image library type (which has had improvements since this request was opened with direct feedback from users) or a more folder-centric approach?
